### PR TITLE
Add Liquid Engine

### LINKED ISSUE #202: Add Liquid Engine Detection
### Some reference to the engine, such as the developer site or a wikipedia article.

https://www.moddb.com/engines/liquid/games

### SteamDB links

https://steamdb.info/app/251530/
https://steamdb.info/app/236730/
https://steamdb.info/app/91200/
https://steamdb.info/app/323190/

### Possible way to detect it

i believe PCTouchHelper.dll is only used in 11bit studios games and Liquid Engine is their inhouse engine so...

### PR DESCRIPTION
**Pull request by diedyesterdaywashere**
(from #202)
### SteamDB app page links to a few games using this

https://steamdb.info/app/91200/
https://steamdb.info/app/236730/
https://steamdb.info/app/251530/
https://steamdb.info/app/252170/
https://steamdb.info/app/282070/
https://steamdb.info/app/294750/
https://steamdb.info/app/323190/

### Brief explanation of the change

Added Detection of Liquid Engine
closes #202

### CODE REVIEW

**Comment by Lifeismana** [comment]
This War of Mine has this file in a folder so you'll have to change the regex rule to also check if it's in a folder (like the other rules)

**Comment by diedyesterdaywashere** [comment]
> This War of Mine has this file in a folder so you'll have to change the regex rule to also check if it's in a folder (like the other rules)

i'll do that in a second :D

**Comment by diedyesterdaywashere** [comment]
> This War of Mine has this file in a folder so you'll have to change the regex rule to also check if it's in a folder (like the other rules)

maybe you could explain how to add multiple checks in one regex? i cant figure it out

**Comment by diedyesterdaywashere** [comment]
whats wrong with it now

**Comment by diedyesterdaywashere** [comment]
why did it reference SteamDatabase#1 lmao

File: rules.ini [hunk 1]
```diff
@@ -83,6 +83,7 @@ idTech7 = \.streamdb$
 Kex = \.kpf$
 KiriKiri = (?:^|/)data\.xp3$
 Lime_OR_OpenFL = (?:^|/)lime(?:-legacy)?\.[hn]dll$
+Liquid = (?:^|)PCTouchHelper\.dll$
```
Comment (Lifeismana) [inline rules.ini:86]:
```suggestion
Liquid = (?:^|/)PCTouchHelper\.dll$
```
almost

**Comment by diedyesterdaywashere** [comment]
@Lifeismana regex101.com was arguing about that slash so i removed it xd

**Comment by diedyesterdaywashere** [comment]
bruh GitHub desktop doesn't want to push stuff now, gives me timed out errors even tho i am online

**Comment by Lifeismana** [comment]
On that site you have to change the delimiter from / to ~ (click the thing just left of the regex)

**Comment by Lifeismana** [comment]
Looks good to me now, thanks for the pr !

**Comment by diedyesterdaywashere** [comment]
thanks for helping me with making this pr too!

**Pull request merged**